src/containers/buttons/ViewBidsButton/ViewBidsButton.tsx

```diff
@@ -0,0 +1,93 @@
+//- Containers Imports
+import { BidList } from 'containers';
+
+//- Library Imports
+import { Bid } from '@zero-tech/zns-sdk/lib/zAuction';
+import { Maybe, Metadata } from 'lib/types';
+import { Domain } from '@zero-tech/zns-sdk/lib/types';
+
+//- Components Imports
+import { FutureButton, Overlay, TextButton } from 'components';
+
+//- Constants Imports
+import { LABELS } from './ViewBidsButton.constants';
+
+interface ViewBidButtonProps {
```

Comment:
Just a thought - we could also make a `ViewBidButton` which just takes a `domainId` as props and gets all the modal data internally. Would cost an extra API call, but could reduce complexity of code as we won't need to pass in a tonne of data. Not necessary for this task though as this seems to be working sweet.